tests/test_training.py

```diff
@@ -86,6 +88,10 @@ class TrainingLoopTests(unittest.TestCase):
     def setUp(self) -> None:
         """Instantiate triples factory and model."""
         self.triples_factory = Nations().training
+        self.random_seed = 123
+        date_string = str(datetime.now()).replace('.', '_').replace(':', '_')
+        self.checkpoint_file = f"PyKEEN_training_loop_test_checkpoint_{date_string}.pt"
```

Comment:
might  be better to use in combination with a temporary directory during the tests

Reply:
https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory